package.json

```diff
@@ -276,7 +276,7 @@
         "react-dom": "^17.0.1",
         "react-draggable": "^3.3.0",
         "react-immutable-proptypes": "^2.1.0",
-        "react-intl": "^3.11.0",
+        "react-intl": "^6.4.2",
```

Comment:
Should this use an exact version to match what we have in our other repositories?